src/tests/remote.test.ts

```diff
@@ -20,7 +20,7 @@ describe('Perl debugger connection', () => {
 	let conn: perlDebuggerConnection;
 
 	beforeEach(() => {
-		conn = new perlDebuggerConnection();
+		conn = new perlDebuggerConnection(null);
```

Comment:
why is it necessary to set `null`?

Reply:
To make explicit that I haven't found a good way to get a `DebugSession` there.

Reply:
I've added a comment another place in the code when solved this one will go away